adafruit_clue.py

```diff
@@ -682,20 +689,20 @@ def pixel(self):
         return self._pixel
 
     @staticmethod
-    def _sine_sample(length):
+    def _sine_sample(length: int):
         tone_volume = (2**15) - 1
         shift = 2**15
         for i in range(length):
             yield int(tone_volume * math.sin(2 * math.pi * (i / length)) + shift)
 
-    def _generate_sample(self, length=100):
+    def _generate_sample(self, length: int = 100):
         if self._audio_out is not None:
             return
         self._sine_wave = array.array("H", self._sine_sample(length))
         self._audio_out = audiopwmio.PWMAudioOut(board.SPEAKER)
         self._sine_wave_sample = audiocore.RawSample(self._sine_wave)
 
-    def play_tone(self, frequency, duration):
+    def play_tone(self, frequency: int, duration: int):
```

Comment:
I think both of these can be `float`.

Reply:
In the docstring it says that the `frequency` is `int`, so I changed `duration` to `float` and kept `frequency` as `int`.